doc/whatsnew/fragments/8181.feature

```diff
@@ -0,0 +1,5 @@
+Add new option (``--show-stdlib``, ``-L``) to pyreverse.
```

Comment:
```suggestion
Add new option (``--show-stdlib``, ``-L``) to ``pyreverse``.
```
Just a nitpick and I know that the existing release notes are also not consistent with this, but the majority of the docs put `pylint`/`pyreverse` in backticks.